docs/apis-clients/operate-api/index.md

```diff
@@ -89,25 +89,27 @@ curl -b cookie.txt -X POST 'http://localhost:8080/v1/process-definitions/search'
+| Endpoint (HTTP verb + URL path)                  |                                                         Description | Notes                                                                                                                                                                                                                                                  |
+| :----------------------------------------------- | ------------------------------------------------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **Process definitions**                          |                                                                     |                                                                                                                                                                                                                                                        |
+| `POST /v1/process-definitions/search`            |                                      Search for process definitions |                                                                                                                                                                                                                                                        |
+| `GET /v1/process-definitions/{key}`              |                                       Get process definition by key |                                                                                                                                                                                                                                                        |
+| `GET /v1/process-definitions/{key}/xml`          |                                Get process definition by key as XML |                                                                                                                                                                                                                                                        |
+| **Process instances**                            |                                                                     |                                                                                                                                                                                                                                                        |
+| `POST /v1/process-instances/search`              |                                        Search for process instances | New fields added: <br/>`flowNodeId`<br/>`flowNodeName`<br/>`processDefinitionKey` <br/><br/>**Warning**<br/>1. New fields could break deserialization, so ignore fields not used.<br/>2. processDefinitionKey only available from version 8.1.8 onward |
```

Comment:
@johanwelgemoed do you think we could explicitly mention that `processDefinitionKey` will be empty for data existed before upgrade to 8.1.8?